app/admin/governance/multitenant_wallet/wallet_admin.py

```diff
@@ -15,7 +15,7 @@
 
 logger = logging.getLogger(__name__)
 
-router = APIRouter(prefix="/admin/wallet-multitenant", tags=["Admin: Wallets"])
+router = APIRouter(prefix="/admin/wallet-multitenant", tags=["admin: wallet"])
 
```

Comment:
I'd say either we do wallets and schemas or wallet and schema? Or did we decide to use singular for one and plural for the other?